doc/development/analyze_your_data.rst

```diff
@@ -0,0 +1,27 @@
+Analyze Your Data
+=================
+
+This section of documentation covers everything you'll need to know to deploy a working Pachyderm cluster and build your own analysis to process whatever data you want. 
```

Comment:
This seems kind of weird to me. This doc is going to cover everything you need to know to deploy which is already a separate doc. I think it's probably better to just link to that doc.

Also how is this doc going to be different from `getting_started.txt` and `beginner_tutorial.rst` it seems like there's going to be a lot of overlap with those.